fix kic image build in prow (no changelog error)

example failure 



https://prow.k8s.io/view/gs/kubernetes-ci-logs/logs/post-minikube-kicbase-image/2053887677656207360


```
0 [linux/arm64 kicbase  8/47] COPY entrypoint /usr/local/bin/entrypoint
#20 CACHED
#21 [linux/arm64 kicbase  9/47] COPY CHANGELOG ./CHANGELOG
#21 ERROR: failed to calculate checksum of ref gf63zvgsatewpjco5715udotq::2nlxi0b7hjik4imfo3wyooz6a: "/CHANGELOG": not found
#10 [internal] load build context
#10 transferring context: 110.63MB 19.4s done
#10 DONE 19.5s
17 skipped lines...
#27 [linux/ppc64le kicbase  6/47] COPY containerd_docker_io_hosts.toml /etc/containerd/certs.d/docker.io/hosts.toml
#27 CACHED
#28 [linux/ppc64le kicbase  9/47] COPY CHANGELOG ./CHANGELOG
#28 ERROR: failed to calculate checksum of ref gf63zvgsatewpjco5715udotq::nddtdvrapbxgl0ivxlzvzh1kd: "/CHANGELOG": not found
#29 [linux/arm64 kicbase  2/47] COPY 10-network-security.conf /etc/sysctl.d/10-network-security.conf
#29 CANCELED
#8 [internal] load build context
14 skipped lines...
  44 |     COPY entrypoint /usr/local/bin/entrypoint
  45 | >>> COPY CHANGELOG ./CHANGELOG
  46 |     COPY nerdctld/nerdctl.socket /usr/local/lib/systemd/system/nerdctl.socket
  47 |     COPY nerdctld/nerdctl.service /usr/local/lib/systemd/system/nerdctl.service
--------------------
ERROR: failed to build: failed to solve: failed to compute cache key: failed to calculate checksum of ref gf63zvgsatewpjco5715udotq::2nlxi0b7hjik4imfo3wyooz6a: "/CHANGELOG": not found
make: *** [hack/prow/prow.mk:99: push-kicbase-image-prow] Error 1
+ EXIT_VALUE=2
+ set +o xtrace
ERROR
ERROR: build step 0 "gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:latest" failed: step exited with non-zero status: 2
BUILD FAILURE: Build step failure: build step 0 "gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:latest" failed: step exited with non-zero status: 2
ERROR: (gcloud.builds.submit) build 68ca2255-59f0-48ef-8788-48eb802e3caf completed with status "FAILURE"
--------------------------------------------------------------------------------
2026/05/11 17:40:27 Failed to run some build jobs: [error running [gcloud builds submit --verbosity info --config /home/prow/go/src/github.com/kubernetes/minikube/hack/prow/image/kicbase/cloudbuild.yaml --substitutions _PULL_BASE_REF=master,_GIT_TAG=v20260511-v1.38.1-102-gf2d1065b9 --project k8s-staging-images --region global --gcs-log-dir gs://k8s-staging-images-gcb/logs --gcs-source-staging-dir gs://k8s-staging-images-gcb/source gs://k8s-staging-images-gcb/source/cef97fbb-8d64-4b0e-aa80-33fa48edd614.tgz --polling-interval 10]: exit status 1]

```

--- a/deploy/kicbase/Dockerfile
+++ b/deploy/kicbase/Dockerfile
@@ -17,7 +17,6 @@
 # For systemd + docker configuration used below, see the following references:
 # https://systemd.io/CONTAINER_INTERFACE/
 
-
 # this ARG needs to be global to use it in `FROM` & is updated for new versions of debian:bookworm-slim-*
 ARG KICBASE_IMAGE="debian:bookworm-20260505-slim"
 # start from debin 12, this image is reasonably small as a starting point

--- a/hack/prow/prow.mk
+++ b/hack/prow/prow.mk
@@ -101,6 +101,7 @@ push-kube-registry-proxy-image-prow:
 
 .PHONY: push-kicbase-image-prow
 push-kicbase-image-prow:
+	./hack/jenkins/build_changelog.sh deploy/kicbase/CHANGELOG
 	./hack/build_auto_pause.sh $(KICBASE_ARCH) $(CURDIR)/deploy/kicbase
 	docker buildx build --push --platform  $(PROW_IMAGE_PLATFORMS) \
 		-t us-central1-docker.pkg.dev/k8s-staging-images/minikube/kicbase:$(_GIT_TAG) -t us-central1-docker.pkg.dev/k8s-staging-images/minikube/kicbase:latest -f deploy/kicbase/Dockerfile deploy/kicbase

